.github/workflows/build.yml

```diff
@@ -35,13 +35,14 @@ jobs:
         - 'device/hid_composite'
         - 'device/hid_composite_freertos'
         - 'device/hid_generic_inout'
+        - 'device/hid_multipleinterface'
```

Comment:
it is a bit late, but `hid_multiple_interface` would be a better name. But no problems, I will do rename later on.